docs/release/release_0_3_0.md

```diff
@@ -0,0 +1,141 @@
+# napari 0.3.0
+
+We're happy to announce the release of napari 0.3.0!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+## Highlights
+
+
+## New Features
+- Hook up reader plugins (#937)
+- Support for magicgui (#981)
+- Writer plugins (#1104)
+
+
+## Improvements
+- Generalize keybindings (#791)
+- Points view data refactor (#951)
+- Add magic name guessing (#1008)
+- Refactor labels layer mouse bindings (#1010)
+- Reduce code duplication for `_on_scale_change` & `_on_translate_change` (#1015)
+- Style refactor (#1017)
+- Add ScaleTranslate transform (#1018)
+- Add docstrings for all the Qt classees (#1022)
+- Sorting and disabling of hook implementations (#1023)
+- Plugin exception storage and developer notification GUI (#1024)
```

Comment:
All of the plugin stuff needs highlighting, not as individual PRs but as a whole.

Reply:
We should indeed include the aics plugin as an example use — and we should reach out to them to update to the new infrastructure.

Reply:
Plugin is now highlighted. @tlambert03 do you want to reach out to Jackson about `napari-plugin-engine`, also to Hector https://github.com/hectormz/napari-mat-images/

Reply:
@sofroniewn remember that new plugin infrastructure is compatible with pluggy, so there is actually no need to update from them. However Jackson already updated napari-aicsimageio! :tada: